xray/commands/audit/generic/generic.go

```diff
@@ -75,15 +75,19 @@ func (auditCmd *GenericAuditCommand) CreateXrayGraphScanParams() *services.XrayG
 }
 
 func (auditCmd *GenericAuditCommand) Run() (err error) {
+	workingDirs, err := xrutils.GetFullPathsWorkingDirs(auditCmd.workingDirs)
```

Comment:
I believe this logic should reside within the inner audit logic since it is applicable only to the CLI context. While it is true that we also address this case in Frogbot, we should not be surprised by this behavior for future features that utilize Audit. (In cases where the working directory is empty, the scan won't automatically run on the root directory.)